Dev james

JKasakyan opened the pull request.

JKasakyan committed:
Merge branch 'skill-huddle/dev'

Commit:
Add suggestion detail page

- Write url, view, html for a page that shows a given suggestion's name, description, netvote, etc.

skill-huddle committed:
Merge pull request #22 from skill-huddle/dev

Two weeks of work

Commit:
Merge remote-tracking branch 'upstream/master' into suggestion-details

skill-huddle committed:
Merge pull request #23 from ian-s-mcb/suggestion-details

Add suggestion detail page

JKasakyan committed:
Add manage league view

— Add url, view, and template for page where head official can manage
league

JKasakyan committed:
Merge branch 'skill-huddle/master' into dev-james

# Conflicts:
#	sh_app/urls.py

ian-s-mcb commented:
Looks good, however I do have two recommendations:
1. The `league_detail` template injects an `li` element outside of its intended parent element. The injected element is a link to the Manage League page. At the moment, the link ends up detached from the navigation/header links. One solution is to edit the `base` template and move start/end block tags for `header` into the `ul` element.
2. A few of our templates contain comments that end up in our html output (ie. what the browser sees). This isn't a huge deal, but it seems like a bad practice to me. Django templates have [their own way](https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#comment) to comment code, and it keeps the html output tidy. The affected templates are `league_detail`, `leagues`, `login`, `manage_league`, and `register`.

ian-s-mcb commented:
Just a test to see that our Slack channel picks up GitHub comments.

JKasakyan committed:
Correct template comments

— Correct placement of manage league link in header
— Originally used html style comments, but to avoid injection into page
when using templates, switched to django template comment tags
